### ISSUE TITLE
Unable to upload file due to permission denied

### ISSUE THREAD

**Issue by ltluc253** [zendesk/sunshine-conversations-android#155]
**For support questions, please visit [our help page](https://smooch.io/help/)**.

This repository's issues are reserved for bug reports only.

 **I'm submitting a ...**
  - [x] bug report
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

 **What is the current behavior?**
- Open Conversation screen
- Tap to + button
- Choose from Photo Gallery or Upload File
- Grant Storage permission
- Select an Image file
- I got Toast message `There was a problem retrieving the photo` and an error below in logcat
```
io.smooch.demoapp E/BitmapFactory: Unable to decode stream: java.io.FileNotFoundException: /storage/emulated/0/DCIM/Camera/IMG_20210413_140823.jpg: open failed: EACCES (Permission denied)
```

 **Please provide the steps to reproduce and, if possible, a minimal demo of the problem**. Ideally a sample project

 **What is the expected behavior?**
- Image or file can be delivered.

 **Please tell us about your environment:**
- SDK version: 8.2.0
- OS version:  Android 10

 **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. Stack Overflow, etc)

**Comment by DataWearsAHood** [comment]
Hello @ltluc253,

Are you able to reproduce the issue with the included /demo app?
Does more than one device demonstrate the same behaviour?

**Comment by ltluc253** [comment]
hi @DataWearsAHood 
The issue can be reproduce in the demo app. I got problem in Android emulator and a physical device. And it happen from version 8.1.0.

**Comment by DataWearsAHood** [comment]
hi @ltluc253, 
Looks like there was a storage-related fix in 8.2.1 - are you still experiencing this issue?
